barretenberg/cpp/src/barretenberg/benchmark/relations_bench/relations.bench.cpp

```diff
@@ -24,7 +24,7 @@ template <typename Flavor, typename Relation> void execute_relation(::benchmark:
     auto params = proof_system::RelationParameters<FF>::get_random();
 
     // Extract an array containing all the polynomial evaluations at a given row i
-    AllValues new_value;
+    AllValues new_value{};
```

Comment:
GCC rightfully caught this is no longer initialized. Plan to do a pass on this with https://github.com/AztecProtocol/barretenberg/issues/793